onnxruntime/python/tools/transformers/models/llama/requirements-cpu.txt

```diff
@@ -1,2 +1,3 @@
 -r requirements.txt
-onnxruntime>=1.16.2
\ No newline at end of file
+onnxruntime>=1.16.2
```

Comment:
Where does this file get used?

Reply:
These `requirements-*.txt` files are used in Anubis, Olive, and inference example notebooks. They also help users know exactly which package versions to install for their scenarios.